test_conformance/c11_atomics/test_atomics.cpp

```diff
@@ -1659,7 +1530,7 @@ template <> double kahan_sub<double>(const std::vector<double> &nums)
     {
         double y = num - compensation;
         double t = sum - y;
-        compensation = (t - sum) - y;
```

Comment:
`(t-sum) == -y`, so `compensation == -2y` here, causing `sum` to explode and overflow to infinity.